src/radical/pilot/agent/scheduler/base.py

```diff
@@ -423,6 +427,8 @@ def slot_status(self):
         Returns a multi-line string corresponding to the status of the node list
         '''
 
+        return '-'
```

Comment:
Too quick and dirty for release?

Reply:
yes -  this only made sense on the large scale experiments we did.